Tests/Marketplace/marketplace_services.py

```diff
@@ -572,7 +572,7 @@ def _parse_pack_metadata(user_metadata, pack_content_items, pack_id, integration
         pack_metadata['certification'] = Pack._get_certification(support_type=pack_metadata['support'],
                                                                  certification=user_metadata.get('certification'))
         pack_metadata['price'] = convert_price(pack_id=pack_id, price_value_input=user_metadata.get('price'))
-        if pack_metadata['price'] > 0:
+        if 'vendorId' in user_metadata:
```

Comment:
`pack_metadata['price'] >= 0` or `'price' in pack_metadata`

Reply:
Both conditions you wrote will always evaluate to `True`.
Isn't the one I wrote correct?

Reply:
Only private packs will have price, no? so it is ok to return True for `pack_metadata['price'] >= 0`
What is the exactly problem then? But your condition is also fine

Reply:
So I prefer to stick with my condition if it's alright with you
We don't validate the fact that `price:0` isn't set in public packs at the moment, and it seems more absolutely correct anyway